coderd/database/querier_test.go

```diff
@@ -3396,7 +3396,6 @@ func TestOrganizationDeleteTrigger(t *testing.T) {
 		require.Error(t, err)
 		// cannot delete organization: organization has 0 workspaces and 1 templates that must be deleted first
 		require.ErrorContains(t, err, "cannot delete organization")
-		require.ErrorContains(t, err, "has 0 workspaces")
```

Comment:
could the number just be changed instead? it seems like we're still counting workspaces, not sure what broke here

Reply:
The change in the postgres function doesn't report resources that are at 0 (or more than 1 for members and groups)